.travis.yml

```diff
@@ -17,14 +17,17 @@ env:
 
 matrix:
     include:
+        # Use the newer stack for HHVM as HHVM does not support Precise anymore since a long time and so Precise has an outdated version
         - php: hhvm
+          sudo: required
+          dist: trusty
         - php: 5.3
-        - php: 5.4
-        - php: 5.5
-        - php: 5.6
-          env: deps=high
-        - php: 7.0
-          env: deps=low
+#        - php: 5.4
+#        - php: 5.5
+#        - php: 5.6
+#          env: deps=high
+#        - php: 7.0
+#          env: deps=low
```

Comment:
This change will be reverted before the merge. It is here to make the build faster until it is working.